drink_machines.lua

```diff
@@ -397,6 +417,9 @@ minetest.register_node('drinks:liquid_barrel', {
       if outputstack == 'vessels:glass_bottle' then
          drinks.drinks_liquid_sub(4, 'jbo', 128, pos)
       end
+      if outputstack == 'vessels:steel_bottle' then
```

Comment:
elseif is clearly my fault, I don't use those much, but I can change that, especially if it improves performance. I'll have to learn about lookup tables, as I've never heard of those, but I think I should be able to implement that as well.